docs/ecology.md

```diff
@@ -1 +1,176 @@
-# Victory
\ No newline at end of file
+# Victory
+
+Victory is a collection of composable React components you can use to build interactive data visualizations.
+
+### Overarching benefits and motivations
+
+1. Bridge the gap between beginners (more likely to use Chart.js or Highcharts) and experts (more likely to use D3)
+  a. Provide a gradient of customization, from none (just pass in data) to full (build your own visualization features)
+  b. Provide interactive documentation to help developers get visualizations into their projects (see [ecology.js](https://github.com/FormidableLabs/ecology)) see [victory-chart](projects.formidablelabs.com/victory-chart/) for an example
+  c. Provide better developer UX generally, for both instantiating data visualizations and creating them
+2. Leverage the power of React to handle DOM, component lifecycle, state, computed styles, etc.
+3. Make data visualizations first class citizens in the open source community
+  a. Repos rather than gists
+  b. forking / issues for continual improvement
+  c. `package.json` for dependencies
+  d. minified dists / umd builds for CDN
+  e. Babel for ES6
+  f. Webpack for builds
+  g. `propTypes` for input validation
+  h. Hot Reloading
+  i. sourcemaps
+  j. `eslintrc`
+  k. Tests
+
+### Using Victory
+
+Victory components can be dropped into your project in seconds. Here's an example, using `victory-pie`.
+
+In your terminal:
+
+`$ npm install victory-pie`
+
+In a view in your React app...
+
+`import {VictoryPie} from "victory-pie";
+
+...
+
+```
+render() {
+  <VictoryPie/>
+}
+
+```
+
+...will render a victory component with default data. The data visualization can then be configured with props...
+
+```
+<VictoryPie
+  endAngle={90}
+  startAngle={-90}
+  innerRadius={140}
+  style={{
+    labels: {
+      padding: -70
+    },
+    data: {
+      stroke: "transparent",
+      opacity: 0.3
+    }
+  }}
+  data={[
+    { x: "Cats", y: 400 },
+    { x: "Dogs", y: 350 },
+    { x: "Frogs", y: 25 },
+    { x: "Turtles", y: 55 },
+    { x: "Chimps", y: 5 }
+  ]}/>
+```
+
+...and completely overridden with child components:
+
+```
+/* where VictoryLabel is your custom label component */
+
+<VictoryPie>
+  <VictoryLabel/>
```

Comment:
label components are not yet supported in pie. I think line would be the most straightforward example here since it's one data element and one label. Other examples that use several labels actually take label components as props because they get mapped over internally, acting more like a prototype for a label. If you want to talk about composing with child components reference chart. default axes vs. custom axes would be a good example